.github/workflows/nightly-tests.yml

```diff
@@ -78,9 +149,9 @@ jobs:
         if: always()
         id: status-message
         run: |
-          L2="Integration: ${{ steps.integration-test.outcome }} "
+          L2="Int: ${{ steps.integration-test.outcome }} "
```

Comment:
I think we need a separator like `  |  ` between each entry for readability in Slack.